### PR TITLE
MGMT-2625 Add HW info to assisted_installer_cluster_hosts metric

### PR DESCRIPTION
**Pull request by nmagnezi**

### CODE REVIEW

**Comment by nmagnezi** [comment]
Tested locally with test-infra, looks okay:
```golang
service_assisted_installer_cluster_hosts{clusterId="306247ce-bb0a-4e9c-932a-2ad50326a164",diskType="HDD",openshiftVersion="4.6",product="KVM",result="Done",role="master",vendor="Red Hat"} 2
```

File: internal/metrics/metricsManager.go [hunk 1]
```diff
@@ -242,10 +242,11 @@ func (m *MetricsManager) ReportHostInstallationMetrics(log logrus.FieldLogger, c
 	}
 }
 
-func (m *MetricsManager) handleHostInstallationComplete(log logrus.FieldLogger, clusterVersion string, clusterID strfmt.UUID, roleStr string, installationStageStr string, h *models.Host) {
+func (m *MetricsManager) handleHostInstallationComplete(log logrus.FieldLogger, clusterVersion string, clusterID strfmt.UUID, roleStr string,
+	hwVendor string, hwProduct string, diskType string, installationStageStr string, h *models.Host) {
 	log.Infof("service Logic Cluster Hosts clusterVersion %s, roleStr %s, result %s",
```
Comment (gamli75) [inline internal/metrics/metricsManager.go:247]: please add the new parameters also to the log

Comment (nmagnezi) [reply internal/metrics/metricsManager.go]: Agreed. Done.

**Review by gamli75 — APPROVED** [review]

**Pull request merged**